.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -232,7 +232,7 @@ class common_job_properties {
                             String buildSchedule = '0 */6 * * *',
                             boolean triggerEveryPush = true,
                             String notifyAddress = 'commits@beam.apache.org',
-                            boolean emailIndividuals = true) {
+                            boolean emailIndividuals = false) {
```

Comment:
+1 for turning it off.
Could this parameter be removed here and from where it is called?

Reply:
Yea, I'm just being minimal because of dynamic typing and inability to easily dry run. I'll go for the full change and just break things perhaps...

Reply:
Done. Gone.